recipe/meta.yaml

```
@@ -9,12 +9,15 @@ source:
   sha256: 041d9eee96445667d2f7b970d2a799592027f1f8818cd96a65dcce1ac0745773
   patches:
     - python_copy_target_fix.patch
-    - setup_py_msvc.patch
     - win_py_install_prefix.patch
+    - osx_cmake.patch
+    - python_flags.patch
+    - osx_link_m.patch
 
 build:
-  number: 3
-  skip: true  # [osx]
```

Could do something like this and re-render to disable the other builds

```suggestion
  skip: true  # [win or linux]
```

---

Thanks, fix applied.  Hope you don't mind the emails for each of my (likely many) subsequent attempts.

---

Ofc. No worries. Get many notifications from conda-forge independent of activity in any one PR :)